client/components/forms/select-opt-groups.jsx

```diff
@@ -1,32 +1,33 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:forms:select-opt-groups' );
+import React, { Component } from 'react';
+import { omit } from 'lodash';
+const debug = require( 'debug' )( 'calypso:forms:select-opt-groups' );
```

Comment:
We shouldn't mix import styles if we can avoid it. In this case, the standard practice would be to import the module and initialize it separately, like:

```js
import debugModule from 'debug';

const debug = debugModule( 'calypso:forms:select-opt-groups' );
```